js/components/portal-dashboard/student-navigator.tsx

```diff
@@ -8,35 +8,38 @@ import { getFormattedStudentName } from "../../util/student-utils";
 interface IProps {
   students: any;
   isAnonymous: boolean;
-  currentStudentIndex: any;
+  currentStudentIndex: number;
   setCurrentStudent: (studentId: string | null) => void;
   currentStudentId: string | null;
-  inResponseDetail?: boolean;
-  key?: any;
+  nameFirst: boolean;
+  filler?: boolean;
 }
 
 export const StudentNavigator: React.FC<IProps> = (props) => {
-  const { inResponseDetail } = props;
-  const componentOrder = inResponseDetail ? [<PrevNextButtons key={"prev-next-button"} {...props}/>, <StudentName key={"student-name"} {...props}/>]
-                                          : [<StudentName key={"student-name"} {...props}/>, <PrevNextButtons key={"prev-next-button"} {...props}/>];
+  const { nameFirst, filler } = props;
+  const componentOrder = nameFirst
+                        ? [<StudentName key={"student-name"} {...props}/>, <PrevNextButtons key={"prev-next-button"} {...props}/>]
+                        : [<PrevNextButtons key={"prev-next-button"} {...props}/>, <StudentName key={"student-name"} {...props}/>];
   return (
-    <div className={css.studentArea}>
-      <div className={css.responseHeader}>
+    <div className={`${css.studentNavigator} ${filler? css.filler : ""}`}>
```

Comment:
spacing:
`<div className={`${css.studentNavigator} ${filler ? css.filler : ""}`}>`

But I'm again voting to remove the filler prop.